test/e2e/framework.go

```diff
@@ -200,6 +214,14 @@ func (f *Framework) afterEach() {
 	f.Client = nil
 }
 
+func (f *Framework) CreateNamespace(baseName string, c *client.Client, labels map[string]string) (*api.Namespace, error) {
+	ns, err := createTestingNS(baseName, c, labels)
```

Comment:
Cleaning up all tests to use `Framework` in #20939. `namespace.go` will require a little bit more refactoring to have it use `f.CreateNamespace`.